.github/workflows/release_driver.yml

```diff
@@ -0,0 +1,111 @@
+name: QLDB Driver Release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*.*.*"
+      
+jobs:
+  build:
+    name: Build
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        os: [windows-latest, macos-latest]
+        
+    steps:
+    - uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+        aws-region: us-east-2
+
+    - uses: actions/checkout@v2
+      with: 
+        submodules: recursive
+        
+    - name: Install dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --configuration Release --no-restore
+    - name: Unit test
+      run: dotnet test Amazon.QLDB.Driver.Tests
+    - name: Integration test
+      run: dotnet test Amazon.QLDB.Driver.IntegrationTests --no-restore --verbosity normal --settings Amazon.QLDB.Driver.IntegrationTests/.runsettings
+
+  release:
+    name: Release
+    needs: [build]
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        dotnet: ['5.0.x']
+      
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_SIGNING_ACCOUNT }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_SIGNING_ACCOUNT }}
```

Comment:
Do we have the secret of `AWS_ACCESS_KEY_ID_SIGNING_ACCOUNT` and `AWS_SECRET_ACCESS_KEY_SIGNING_ACCOUNT`?